src/user/index.ts

```diff
@@ -217,13 +217,61 @@ export const getUser = async (uuid: string) => {
 			...(await Oauth.getAuthHeaders())
 		}
 	})
-	if(result.status !== 200) {
-		throw new AuthError("Unable to get user");
+	if(result.status === 403) {
+		throw new AuthError("You do not have access to view this user. (403 Forbidden)");
+	}
+	else if(result.status !== 200) {
+		throw new AuthError("We were unable to get information about this user.");
```

Comment:
why is everything but 200 an auth error?

Reply:
Changed to `ApiGetError` instead!